api/src/test/java/org/openmrs/module/mksreports/reports/OutpatientConsultationReportManagerTest.java

```diff
@@ -89,40 +89,47 @@ public void testReport() throws Exception {
 			// one row
 			
 			// Ensure that the report contains 4 possible combinations
-			Cohort col1 = (Cohort) row.getColumnValue("MALARIA." + OutpatientConsultationReportManager.col7);
-			assertNotNull(col1);
-			assertEquals(1, col1.getSize());
-			Cohort col2 = (Cohort) row.getColumnValue("MALARIA." + OutpatientConsultationReportManager.col12);
-			assertNotNull(col2);
-			assertEquals(0, col2.getSize());
-			Cohort col3 = (Cohort) row.getColumnValue("FEVER." + OutpatientConsultationReportManager.col7);
-			assertNotNull(col3);
-			assertEquals(1, col3.getSize());
-			Cohort col4 = (Cohort) row.getColumnValue("FEVER." + OutpatientConsultationReportManager.col12);
-			assertNotNull(col4);
-			assertEquals(0, col4.getSize());
+			Cohort fiveYearsTo15YearsMalesWithMalaria = (Cohort) row
```

Comment:
Great idea, however let's try to compact those names more →
```java
_5To15yMalesWithMalaria
```
maybe even
```java
_5To15yMalesMalaria
```
Prefixing variables names with `_` is quite a restricted practise, but let's do it in this case. Can you also do this in the actual code base for cohort definitions? See [here](https://github.com/mekomsolutions/openmrs-module-mksreports/blob/db8e28ec9b13ca0ab5d66943503547dfe27d7903/api/src/main/java/org/openmrs/module/mksreports/reports/OutpatientConsultationReportManager.java#L170) and below.

Reply:
After doing the refactoring something like `zeroTo1Month` now looks like `_0To1m` don't know if it's just me but the latter reads more like zero to 1 million. I mean that's the impression I get when I look at it.